src/atomic_ptr.hpp

```diff
@@ -34,7 +34,8 @@
 #define ZMQ_ATOMIC_PTR_MUTEX
 #elif defined ZMQ_HAVE_ATOMIC_INTRINSICS
 #define ZMQ_ATOMIC_PTR_INTRINSIC
-#elif (defined __cplusplus && __cplusplus >= 201103L)
+#elif (defined __cplusplus && __cplusplus >= 201103L                           \
```

Comment:
might be better with a couple more brackets for easier reading

Reply:
Ack. Will add them.